service/openapi.yaml

```diff
@@ -12,6 +12,27 @@ security:
   - ApiKeyAuth: [ ]
   - BearerAuth: [ ]
 paths:
+  /ofrep/v1/metadata:
```

Comment:
This type of endpoint is an excellent example on when caching can be used. Let's have that included as well 👍

Reply:
Since it will called only during initialization that's why I had not included any caching.

What did you had in mind? The same mechanism that we have for the ` /ofrep/v1/evaluate/flags` endpoint _(with a 304 response + a management of the `If-none-match`)_?

Reply:
Yes exactly like the others, etag + if-none-match.
Maybe it doesn't make much sense in "volatile" environments like JS, but for mobile environments the http caching is often persisted on disk and then it would be good to have.

Reply:
I've added the support of etag + if-none-match.